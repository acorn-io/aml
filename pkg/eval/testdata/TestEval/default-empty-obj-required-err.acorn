define empty: {
    obj: object
}

empty({
})
