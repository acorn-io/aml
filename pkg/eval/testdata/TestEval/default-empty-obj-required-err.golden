{
  "empty": {
    "type": "object",
    "properties": {
      "obj": {
        "type": "object"
      }
    }
  },
  "obj": {}
}